pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -418,6 +418,7 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 			}
 			// Use the termination message written by the application is not empty
 			if len(tMessage) != 0 {
+				klog.Info("PONTIYA   ======================   STATUS ======= LOG  :  ", tMessage, "--Here-------------")
```

Comment:
Did you mean to leave this message in here?  It looks like debug code?

Reply:
No, I will remove it.